Sources/Swift/Tools/UIRedactBuilder.swift

```diff
@@ -248,14 +264,22 @@ class UIRedactBuilder {
      Indicates whether the view is opaque and will block other view behind it
      */
     private func isOpaque(_ view: UIView) -> Bool {
-        return  view.alpha == 1 && view.backgroundColor != nil && (view.backgroundColor?.cgColor.alpha ?? 0) == 1
+        return SentryRedactViewHelper.shouldClipOut(view) || (view.alpha == 1 && view.backgroundColor != nil && (view.backgroundColor?.cgColor.alpha ?? 0) == 1)
     }
 }
 
 @objcMembers
-class SentryRedactViewHelper: NSObject {
+public class SentryRedactViewHelper: NSObject {
```

Comment:
I’m not super happy about making this class public, but SentrySwiftUI is an external module that needs access to it. Also, it’s impossible to misuse it in a way that would cause a crash.

Reply:
I see that you added some new methods. Maybe you can create an extra `SentryRedactViewHelper` just for SentrySwiftUI so you don't need to make it public?

Reply:
Not possible, I need to read it from RedactBuilder class that needs to be in the Sentry SDK.
Maybe if we split SR into a different module already we would need to worry less about this being public